src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -512,6 +529,19 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
                     out = {"tokens": tokens["sequences"], "token_timestamps": token_timestamps}
             else:
                 out = {"tokens": tokens}
+
+            if "prompt_ids" in generate_kwargs:
```

Comment:
This should be a post-processing step, rather than in the `_forward`. Could you move it to the method `postprocess` please?

Reply:
@sanchit-gandhi , Thank you so much for the review. will work on your comment.